analyses/mdg/mdg_plague_exploration.md

```diff
@@ -184,6 +184,11 @@ We had an initital dataset and later received an updated one. Compare these two
 As can be seen in the graph, the cases in 2021 are slightly smoothed in the new dataset, which can be caused by newly available information. 
 Numbers for previous years also changed for some dates, and we are thus far unclear why this occurred. 
 
+```python
+plague_data_filename_old = "Madagascar_IPM_Plague_cases_Aggregated_2021-09-24.csv"
```

Comment:
Would it be better to define these file paths at the top of the script?

Reply:
Yes, makes sense